restapi/src/test/java/io/stargate/web/docsapi/service/DocumentServiceTest.java

```diff
@@ -527,6 +528,108 @@ void testGetDocPathFields() throws JsonProcessingException {
         .isEqualTo("[{\"a\":{\"d\":{\"c\":3,\"z\":true}}},{\"a\":{\"f\":{\"c\":100}}}]");
   }
 
+  @Test
+  void testGetDocPathPlainFields() throws JsonProcessingException {
+    final String id = "id2";
+    withQuery(table, selectAll("WHERE key = ? AND p0 = ? AND p1 = ?"), id, "a", "d")
+        .returning(
+            ImmutableList.of(
+                row(id, 3.0, "a", "d", "c"),
+                row(id, true, "a", "d", "z"),
+                row(id, "zz", "a", "d", "y")));
```

Comment:
I see this in other places but what is this doing? Should we assert something on the returned `io.stargate.db.datastore.ValidatingDataStore.QueryExpectation`?

Reply:
Query expectations are automatically validated in `@AfterEach AbstractDataStoreTest.checkExpectedExecutions()`.

Is this what you mean?

Reply:
Ah, missed that. My bad

Reply:
As for `QueryAssert.assertExecuteCount()` I do not think it is critical for this test since we automatically assert as least one execution.